okio/src/jvmTest/java/okio/WaitUntilNotifiedTest.java

```diff
@@ -56,7 +56,6 @@
     double start = now();
     try {
       timeout.waitUntilNotified(this);
-      fail();
     } catch (InterruptedIOException expected) {
```

Comment:
I think we might want to increase the tolerance here; I definitely don't want to get rid of this fail() call.

Reply:
I'm not sure what you mean by "tolerance" ? We could keep the `fail()` for non-Windows hosts but on windows the early notification isn't caught by `waitUntilNotified()` so not sure how to test that.

Reply:
Let's figure that out in the other PR. For this you can just do assumeNotWindows before the test proceeds so we get a green build.